.github/workflows/build.yml

```diff
@@ -0,0 +1,114 @@
+name: Build & Integration tests
+
+on:
+  push:
+    branches:
+      - main
+      # For testing this github Action's setup
+      - feat/ios-update
+  workflow_dispatch:
+
+concurrency:
+  group: release-build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  unit-tests:
+    runs-on: macos-13 # or macos-latest if you prefer, but be aware that the available simulators could be different if you run a different version
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Flutter SDK
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+          # instead of "channel: stable", you could be more precise by specifying the exact version of Flutter you're using:
+          # flutter-version: '<FLUTTER_VERSION>'
+
+      - name: Install Flutter dependencies
+        run: flutter pub get
+
+      - name: Flutter lint
+        run: flutter analyze
+
+      - name: Run unit tests
+        run: flutter test test
```

Comment:
Don't we also need to check the `flutter build`?
Also, check the lint in the example app and build.

Reply:
for the Flutter SDK, it will not compline but is included as source code to the SDK's consumer app. We haven't `flutter build` for the package. If we have a syntax error or missing dependency, the `flutter analyze` and `test` will fail.

Reply:
I will add one more pipeline for checking the example app.
It will only run when any file on the example app changed

Reply:
I added a lint check for the example app.
For building the example app,  I only enabled check to build the Android version.
https://github.com/duyhungtnn/bkt-flutter/pull/1/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R131

For the iOS version, it needed an Apple developer account. Adding an Apple developer account, may not be necessary for an SDK repo.
<img width="1269" alt="Screenshot 2023-07-21 at 16 37 43" src="https://github.com/duyhungtnn/bkt-flutter/assets/2597710/624823f8-7b85-4f36-b54a-69010fced186">